admin/src/components/Dashboard/Widgets/ConsentByProcessor.test.js

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+import ConsentByProcessor from './ConsentByProcessor';
+import toJson from 'enzyme-to-json';
+import ItemsCard from 'components/core/cards/dashboard/ItemsCard';
+
+describe('(Component) ConsentByProcessor', () => {
+  it('should render', () => {
+    const props = {
+      totalSubjects: 100,
+      processors: [
+        {
+          name: 'p1',
+          consented: 1
+        }
+      ]
+    };
+    const component = shallow(<ConsentByProcessor {...props} />);
+    expect(toJson(component)).toMatchSnapshot();
+  });
+
+  it('should render if numbers are 0', () => {
```

Comment:
This looks like an exact copy of the test above. Is there a difference?

Reply:
I made this test actually test the 0 case now 👍